CodeSniffer/Standards/PEAR/Sniffs/NamingConventions/ValidClassNameSniff.php

```diff
@@ -42,6 +42,7 @@ public function register()
         return array(
                 T_CLASS,
                 T_INTERFACE,
+                T_TRAIT,
```

Comment:
The PEAR standard was written before many language features, like traits, so I keep it updated from time to time in this same way. I don't see any problem with this change.